src/main/java/io/mosip/mimoto/constant/SwaggerLiteralConstants.java

```diff
@@ -0,0 +1,77 @@
+package io.mosip.mimoto.constant;
+
+public class SwaggerLiteralConstants {
+
+    /*  Attestation Controller */
+    public static final String ATTESTATION_NAME = "Attestation" ;
+    public static final String ATTESTATION_DESCRIPTION = "All the Attestation Related Endpoints" ;
+
+
+    /*  Common Inji Controller */
+    public static final String COMMON_INJI_NAME = "Properties" ;
+    public static final String COMMON_INJI_DESCRIPTION = "All the Inji Wallet Properties Related Endpoints" ;
+    public static final String COMMON_INJI_GET_PROPERTIES_SUMMARY = "Retrieve All Inji Wallet Properties" ;
+    public static final String COMMON_INJI_GET_PROPERTIES_DESCRIPTION = "This endpoint allow you to retrieve all the Inji Wallet Properties" ;
+
+    /*  Credentials Controller */
+    public static final String CREDENTIALS_NAME = "Credentials Download using OpenId4VCI" ;
+    public static final String CREDENTIALS_DESCRIPTION = "All the Credentials Related Endpoints" ;
+    public static final String CREDENTIALS_DOWNLOAD_VC_SUMMARY = "Download Credentials as PDF" ;
+    public static final String CREDENTIALS_DOWNLOAD_VC_DESCRIPTION = "This endpoint allow you to retrieve all the Inji Wallet Properties" ;
```

Comment:
description is not right, it has to be about download credential as PDF

Reply:
Updated